phpunit.xml.dist

```diff
@@ -14,8 +14,8 @@
         <env name="DB_HOST" value="127.0.0.1"/>
         <env name="DB_PORT" value="3306"/>
         <env name="DB_DATABASE" value="laravel_excel"/>
-        <env name="DB_USERNAME" value="root"/>
-        <env name="DB_PASSWORD" value=""/>
+        <env name="DB_USERNAME" value="esupl"/>
```

Comment:
Can you revert this? Tip for local development: you can copy `phpunit.xml.dist` to `phpunit.xml`, the last one is gitignored